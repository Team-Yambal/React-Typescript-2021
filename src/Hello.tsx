import * as React from "react";

const Hello: React.FC = () => <h1>おはようございます！！</h1>;

export default Hello;
