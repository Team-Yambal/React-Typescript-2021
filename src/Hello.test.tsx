import React from "react";
import renderer from "react-test-renderer";
import Hello from "./Hello";

test("renders learn react link", () => {
  const component = renderer.create(<Hello />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
