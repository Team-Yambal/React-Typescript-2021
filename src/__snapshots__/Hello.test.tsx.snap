// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<h1>
  おはようございます！！
</h1>
`;
